docs/src/pages/theme/theme-merge.mdx

```diff
@@ -0,0 +1,109 @@
+import { Link } from 'gatsby';
+import { ThemeExplorer } from '../../components/ThemeExplorer';
+
+# themeMerge()
```

Comment:
this is probably to late for this. But i don't love the function name `themeMerge`. I view merges as forming the union of two mostly disjoint sets, with undefined behavior for what happens to the set of things in the overlap. This function in my head should be called `themeEditor` or `themeSetter`. We take in a set of settings to override the default root theme configuration.

Reply:
For the most part, `themeMerge` is an object merge utility that defaults to using `RootTheme`. Because it follows similar patterns as other merge utilities I chose its name to reflect that. I feel like it could go either way, but I lean more towards 'merge' for that reason.

Reply:
kewl beans